static/app/views/dashboardsV2/datasetConfig/releases.tsx

```diff
@@ -11,7 +21,36 @@ export const ReleasesConfig: DatasetConfig<
   transformSeries: (_data: SessionApiResponse | MetricsApiResponse) => {
     return [] as Series[];
   },
-  transformTable: (_data: SessionApiResponse | MetricsApiResponse) => {
-    return {data: []} as TableData;
-  },
+  transformTable: transformSessionsResponseToTable,
 };
+
+function transformSessionsResponseToTable(
```

Comment:
This function used to live in a dedicated file named after itself, should we be moving those functions over into here? I know this one has slightly different logic because it parses the derived status fields instead of having them passed in, so I wonder if there are any dependencies on the old function and if not, can we remove it?

Reply:
gonna punt this till we have some more functions and see if there's a good way to organize them